tests/12login/01threepid-and-password.pl

```diff
@@ -120,24 +120,11 @@ =head2 add_email_for_user
 Add the given email address to the homeserver account, including the
 verfication steps.
 
-C<%params> may include:
-
-=over
-
-=item bind => SCALAR
-
-If truthy, we will ask the server to ask the IS to bind the email
-address. False by default.
-
-=back
-
 =cut
 
 sub add_email_for_user {
```

Comment:
If `bind_email_for_user` is using the new bind API, should this method use the new add API?

Reply:
I guess... we can do that later though?

Reply:
True